client/signup/signup-header/index.tsx

```diff
@@ -44,27 +43,41 @@ const SignupHeader = ( {
 	);
 	const showProgressBar = progressBar.flowName !== FREE_FLOW;
```

Comment:
Just a side note that the progress bar that used to be shown on Stepper flows is no longer there in production; not sure if it's intentional 🤔

Reply:
@southp Quite an on-point observation I think. 

Also indirectly leads me to another (maybe obvious) deviation between Stepper & Start -> the latter controlling the loading screen via `shouldShowLoadingScreen`, and it happens within the current step, while the former doing it separately in another step (`processing`). So I think we are good in the logic as the issue with the lingering banner shouldn't be an issue beyond Start flows.

Reply:
> So I think we are good in the logic as the issue with the lingering banner shouldn't be an issue beyond Start flows.

However we'd still need to pass `progressBar` through as it's currently an empty object in Stepper. and maybe worth renaming, or ditching altogether in favor of separate `flowName`/`stepName` props.